cli/src/semgrep/core_runner.py

```diff
@@ -817,8 +818,17 @@ def _run_rules_direct_to_semgrep_core_helper(
             # TODO: use exact same command-line arguments so just
             # need to replace the SemgrepCore.path() part.
             if deep:
-                logger.error("!!!This is a proprietary extension of semgrep.!!!")
-                logger.error("!!!You must be logged in to access this extension!!!")
+                if auth.get_token() is None:
+                    logger.error("!!!This is a proprietary extension of semgrep.!!!")
+                    logger.error("!!!You must be logged in to access this extension!!!")
+                else:
+                    logger.error(
```

Comment:
Should this be a `logger.error` ? It doesn't look like an error.

Reply:
it should not